test/e2e/pod_infra_container_test.go

```diff
@@ -95,7 +94,7 @@ var _ = Describe("Podman pod create", func() {
 		session.WaitWithDefaultTimeout()
 		Expect(session.ExitCode()).To(Equal(0))
 
-		check := podmanTest.Podman([]string{"ps", "-a", "--no-trunc", "--ns", "--format", "{{.IPC}} {{.NET}}"})
+		check := podmanTest.Podman([]string{"ps", "-a", "--no-trunc", "--ns", "--format", "{{.Namespaces.IPC}} {{.Namespaces.NET}}"})
```

Comment:
The "Namespaces" changes seem like a breaking change. Is this known/intended? @mheon @baude

Reply:
the change is intended, though we should probably use methods to keep backwards compat?

Reply:
+1 ... then we can make sure to test the new and old format here

Reply:
good idea.  I've added methods to keep compatibility and fixed the tests to use both versions